ca/testutils/staticcerts.go

```diff
@@ -23,17 +23,17 @@ AwEHoUQDQgAEMLSpaZt8CRkfTXFRWPDBH7ai+i1TWJUhQPZ84Wb3dZR+DfVzKBr1
 -----END EC PRIVATE KEY-----
 `)
 
-	// ExpiredCert is an ECDSA certificate that expired in 2007
```

Comment:
The test in this PR no longer uses this particular cert, but while I was trying stuff I discovered that I had set the validity times wrong for this cert:  it was previously valid from 1987-1977 (not a valid date range).  So I regenerated this one.